bin/stress-test/src/store/mod.rs

```diff
@@ -41,10 +41,10 @@ pub async fn bench_sync_state(data_directory: PathBuf, iterations: usize, concur
 
     let (store_client, _) = start_store(data_directory).await;
 
-    // each request will have 3 account ids, 3 note tags and will be sent with block number 0
+    // each request will have 5 account ids, 5 note tags and will be sent with block number 0
     let request = |_| {
         let mut client = store_client.clone();
-        let account_batch: Vec<AccountId> = account_ids.by_ref().take(3).collect();
+        let account_batch: Vec<AccountId> = account_ids.by_ref().take(5).collect();
         tokio::spawn(async move { sync_state(&mut client, account_batch, 0).await })
```

Comment:
I would create constants for these numbers so that they are easy to adjust in the future.